tests/mlmodel_openai/conftest.py

```diff
@@ -37,7 +39,7 @@
     "debug.log_data_collector_payloads": True,
     "debug.record_transaction_failure": True,
     "machine_learning.enabled": True,
```

Comment:
I believe I removed this in the sync PR but we probably don't need the `machine_learning.enabled` check here.